app/routes/main.js

```diff
@@ -3,18 +3,28 @@ import config from 'travis/config/environment';
 
 export default TravisRoute.extend({
   renderTemplate() {
-    $('body').attr('id', 'home');
```

Comment:
I really don't like this style of differentiating between layouts, but I think that we need to leave it here. We rely on the id of a body element in a few places and if it doesn't get set here, we may end up with an unexpected result. We don't use `#home` anywhere in the sass from what I see, but we use `#not-found` (among others). So for example when you go to "not found" page and transition to home page, you would end up with the wrong id.